extensions/ui/org.eclipse.smarthome.ui.paper/web-src/js/controllers.configuration.js

```diff
@@ -414,21 +416,68 @@ angular.module('PaperUI.controllers.configuration',
         }
     };
     
-    $scope.getChannels = function(advanced) {
+    var getChannels = function(advanced) {
+    	
         if (!$scope.thingType || !$scope.thing) {
             return;
-        }
-        return $.grep($scope.thing.channels, function(channel, i) {
-           var channelType = $scope.getChannelTypeById(channel.id);
-           return channelType ? advanced == channelType.advanced : false;
-        });
+        }   
+        var thingChannels=[];
+		var includedChannels = [];
+		$scope.isAdvanced = filterAdvance($scope.thing.channels, true).length>0;
+		if ($scope.thingType.channelGroups && $scope.thingType.channelGroups.length > 0) {
+			for (var i = 0; i < $scope.thingType.channelGroups.length; i++) {
+				var group = {};
+				group.name = $scope.thingType.channelGroups[i].label;
+				group.description = $scope.thingType.channelGroups[i].description;
+				group.channels = matchGroup($scope.thing.channels, $scope.thingType.channelGroups[i].id);
+				group.channels = advanced ? group.channels : filterAdvance(group.channels, false);
+				includedChannels = includedChannels.concat(group.channels);
+				thingChannels.push(group);
+			}
+			var group = {"name" : "Others","description" : "Other channels","channels":[]};
+			for (var i = 0; i < $scope.thing.channels.length; i++){
+				if(includedChannels.indexOf($scope.thing.channels[i])==-1)
+					group.channels.push($scope.thing.channels[i]);
+			}
+			if(group.channels && group.channels.length > 0){
+				thingChannels.push(group);
+			}
+		} else {
+			var group = {};
+			group.name = "_default";
```

Comment:
No, '_default' group means that no groups were found so all channels should be displayed without any grouping. I've removed the name also as it could have got confusing, if the groups also had a group with name '_default'.